src/USER-SICRACK/fix_sicrack.cpp

```diff
@@ -0,0 +1,318 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   www.cs.sandia.gov/~sjplimp/lammps.html
+   Steve Plimpton, sjplimp@sandia.gov, Sandia National Laboratories
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under 
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+   //***CETTINA
+   //Mode 1 crack
+/*------------------------------------------------------------------------- */
+
+#include "math.h"

```

Comment:
this file violates several conventions for how we format and order include statements. please see the corresponding discussion of the policy in https://github.com/lammps/lammps/blob/master/.github/CONTRIBUTING.md#contributing-code and https://github.com/lammps/lammps/blob/master/doc/include-file-conventions.md